artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -0,0 +1,1513 @@
+        "403":
+          $ref: '#/components/responses/Generic403'
+        "409":
+          $ref: '#/components/responses/Generic409'
+        "500":
+          $ref: '#/components/responses/Generic500'
+        "503":
+          $ref: '#/components/responses/Generic503'
+    get:
+      tags:
+        - event-type notification subscription
+      summary: 'Retrieve a list of event-type event subscription'
+      description: Retrieve a list of event-type event subscription(s)
+      operationId: retrieveSubscriptionList
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
+      security:
+        - openId:
+          - event-type:subscriptions:read
+      responses:
+        "200":
+          description: List of event subscription details
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Subscription'
+        "400":
+          $ref: '#/components/responses/Generic400'
+        "401":
+          $ref: '#/components/responses/Generic401'
+        "403":
+          $ref: '#/components/responses/Generic403'
+        "500":
+          $ref: '#/components/responses/Generic500'
+        "503":
+          $ref: '#/components/responses/Generic503'
+  /subscriptions/{subscriptionId}:
+    get:
+      tags:
+        - event-type notification subscription
+      summary: 'Retrieve a event-type event subscription'
+      description: retrieve event subscription information for a given subscription.
+      operationId: retrieveSubscription
+      security:
+        - openId:
+          - event-type:subscriptions:read
+      parameters:
+        - $ref: "#/components/parameters/SubscriptionId"
+        - $ref: '#/components/parameters/x-correlator'
+      responses:
+        "200":
+          description: OK
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Subscription'
+        "400":
+          $ref: '#/components/responses/SubscriptionIdRequired'
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/Generic404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "503":
+          $ref: "#/components/responses/Generic503"
+    delete:
+      tags:
+        - event-type notification subscription
+      summary: 'Delete a event-type event subscription'
+      operationId: deleteSubscription
+      description: delete a given event subscription.
+      security:
+        - openId:
+          - event-type:subscriptions:delete
```

Comment:
same as https://github.com/camaraproject/Commonalities/pull/189/files#r1575800606

Reply:
Yes same
I should have here only scope `event-type-notification-subscriptions:delete`